hillel_django/settings.py

```diff
@@ -220,6 +225,18 @@
             'access_type': 'offline',
         },
     },
+    'telegram': {
+        'SCOPE': ['basic_profile'],
+        'AUTH_PARAMS': {'access_type': 'offline'},
+        'METHOD': 'oauth2',
+        'VERIFIED_EMAIL': True
+    },
+    'twitter': {
+        'APP': {
+            'consumer_key': 'bmnY8bXbE0gmXNyK7mclCC46c',
+            'secret': 'dDZfodapwTDPlzXkbX9PN0msnJ461V1cUXDVrjoXErZnDz0oXp',
```

Comment:
the secret should be kept in environment variable (.env), never share it in your code

Reply:
fixed it with new commit